db/import-tmpl/item_enchant.yml

```diff
@@ -0,0 +1,71 @@
+# This file is a part of rAthena.
+#   Copyright(C) 2022 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Item Enchant Database
+###########################################################################
+#
+# Item Enchant Settings
+#
+###########################################################################
+# - Id                         Clientside LUA index
+#   TargetItems:               List of possible target item(s)
+#     <item name>              Item name of the possible target item (true/false)
+#   MinimumRefine              Minimum refine of the target item (default: 0)
+#   MinimumEnchantgrade        Minimum enchantgrade of the target item (default: 0)
+#   AllowRandomOptions         Are random options in the target item allowed? (Default: true)
+#   Reset:                     Reset options (Default: disabled)
+#     Chance                   Chance of successful reset (Default: 0)
+#     Price                    Price for trying to reset the enchants (Default: 0)
+#     Materials:               Items required for trying to reset the enchants (Default: none)
+#       - Material             Item name of the required item
+#         Amount               Amount of the required item (Default: 1)
+#   Order:                     Order in which the slots can be enchanted (Default: none)
+#     - Slot                   Number of the slot (0-3)
+#   Slots:                     Enchant options (Default: none)
```

Comment:
Default can be removed since the parser requests the field
https://github.com/rathena/rathena/pull/7186/files#diff-7d714f7dc5f324f89ba843ffae51f2bc5eecd8588c55dec3976fda1efc2451d1R2013
```suggestion
#   Slots:                     Enchant options
```